protos/sources/vega/commands/v1/commands.proto

```diff
@@ -132,6 +160,20 @@ message LiquidityProvisionAmendment {
   string reference = 6; // empty string means no change
 }
 
+// Amend a liquidity provision request
+message SpotLiquidityProvisionAmendment {
+  // Unique ID for the market with the liquidity provision to be amended.
+  string market_id = 1;
+
+  // From here at least one of the following is required to consider the command valid.
+  string buy_commitment_amount = 2; // empty strings means no change
+  string sell_commitment_amount = 4; // empty strings means no change
+  string fee = 3; // empty strings means no change
+  repeated vega.LiquidityOrder sells = 5; // empty slice means no change
+  repeated vega.LiquidityOrder buys = 6; // empty slice means no change
+  string reference = 7; // empty string means no change
```

Comment:
Should we be able to update the reference?

Reply:
I think we can for futures?